Formula/alerter.rb

```diff
@@ -0,0 +1,23 @@
+class Alerter < Formula
+  desc "Send User Alert Notification on Mac OS X from the command-line"
+  homepage "https://github.com/vjeantet/alerter"
+  url "https://github.com/vjeantet/alerter/archive/refs/tags/004.tar.gz"
+  version "4"
+  sha256 "c4c16735e1a57ce04a5acfa762bd44ab8ef10884308725ab886b984b1de17bfe"
+  license "MIT"
+
+  depends_on xcode: :build
+
+  def install
+    project_file = "alerter.xcodeproj/project.pbxproj"
+    old_identity = "CODE_SIGN_IDENTITY = \"Apple Development\";"
+    new_empty_identity = "CODE_SIGN_IDENTITY = \"-\";"
+    inreplace project_file, old_identity, new_empty_identity
+    xcodebuild "-project", "alerter.xcodeproj", "-target", "alerter", "-configuration", "Release", "SYMROOT=build"
```

Comment:
```suggestion
    xcodebuild, "-arch", Harware::CPU.arch,
               "-project", "alerter.xcodeproj",
               "-target", "alerter",
               "-configuration", "Release",
               "SYMROOT=build",
               "CODE_SIGN_IDENTITY="
```
The `-arch` flag should fix the CI failure (I think), and I think we can also do away with the `inreplace` in favour of of passing `CODE_SIGN_IDENTITY=` to `xcodebuild`. Try this out locally to see if it works first...